sdcm/utils/benchmarks.py

```diff
@@ -60,13 +103,14 @@ def __init__(self, node: "BaseNode"):
         self._remoter: RemoteCmdRunnerBase = node.remoter
 
     def install_benchmark_tools(self) -> bool:
-        self._clone_sysbench_repo()
+        self._clone_repo("https://github.com/akopytov/sysbench.git")
```

Comment:
This repo path is written more than 1 time in different places. It is not correct.
Please, reuse one single place for it. Moreover, the repo you use is personal for some guy. 
It is better to use our own clone which gets synced periodically.

Reply:
Fixed with rebase